pkgs/apps/gpaw/SetupPath.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/gpaw/__init__.py b/gpaw/__init__.py
+index b5c029e13..16054dacd 100644
+--- a/gpaw/__init__.py
++++ b/gpaw/__init__.py
+@@ -198,15 +198,7 @@ def read_rc_file():
+ 
+ 
+ def initialize_data_paths():
+-    try:
+-        setup_paths[:0] = os.environ['GPAW_SETUP_PATH'].split(os.pathsep)
+-    except KeyError:
+-        if len(setup_paths) == 0:
+-            if os.pathsep == ';':
+-                setup_paths[:] = [r'C:\gpaw-setups']
+-            else:
+-                setup_paths[:] = ['/usr/local/share/gpaw-setups',
+-                                  '/usr/share/gpaw-setups']
++    setup_paths[:0] = @gpawSetupPath@
```

Comment:
Maybe a little less brute force? Somthing like this:
```suggestion
try:
    setup_paths[:0] = os.environ['GPAW_SETUP_PATH'].split(os.pathsep)
except KeyError:
        if len(setup_paths) == 0:
            setup_paths[:0] = @gpawSetupPath@
```

That way you can still use `GPAW_SETUP_PATH` as intended by gpaw, but you also have the built default.

Reply:
We can then also remove the `preCheck` phase?

Reply:
Yep, very good idea. Changed it.